install/zigbee2mqtt-install.sh

```diff
@@ -14,31 +14,31 @@ network_check
 update_os
 
 msg_info "Installing Dependencies"
-$STD apt-get install -y curl
-$STD apt-get install -y sudo
-$STD apt-get install -y mc
-$STD apt-get install -y git
-$STD apt-get install -y make
-$STD apt-get install -y g++
-$STD apt-get install -y gcc
-$STD apt-get install -y ca-certificates
-$STD apt-get install -y gnupg
-msg_ok "Installed Dependencies"
+$STD apt-get install -y curl \
+   sudo \
+   mc \
+   git \
+   make \
+   g++ \
+   gcc \
+   ca-certificates \
+   gnupg
+msg_ok "Installing Dependencies"
```

Comment:
```suggestion
msg_ok "Installed Dependencies"
```
What are you basing all these message changes on? They already look correct to me